src/crl.rs

```diff
@@ -342,22 +342,147 @@ fn parse_revoked_cert<'a>(der: &mut untrusted::Reader<'a>) -> Result<RevokedCert
 
         let revocation_date = der::time_choice(der)?;
 
+        let mut revoked_cert = RevokedCert {
+            serial_number,
+            revocation_date,
+            reason_code: None,
+            invalidity_date: None,
+        };
+
         // RFC 5280 §5.3:
         //   Support for the CRL entry extensions defined in this specification is
         //   optional for conforming CRL issuers and applications.  However, CRL
         //   issuers SHOULD include reason codes (Section 5.3.1) and invalidity
         //   dates (Section 5.3.2) whenever this information is available.
         if !der.at_end() {
-            // Consume extension bytes without parsing.
-            // TODO(@cpu): Parse these extensions.
-            let _ = der.read_bytes_to_end();
+            // It would be convenient to use der::nested_of_mut here to unpack a SEQUENCE of one or
```

Comment:
Style nit: in order to avoid rightward drift, I'd probably invert the condition here to return early (and maybe do the same for the `ext_seq.is_empty()` check)?

Reply:
Done :+1: